Creating-Objects.md

```diff
@@ -28,10 +28,12 @@ When collecting user input, we often define the kind of input we expect using [[
 The schema would match a dictionary like this:
 
 ```json
-{"version": 34}
+{"length": 34.5}
 ```
 
-This schema describes a dictionary with a `version` key whose value is an `int`, and the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
+This schema describes a dictionary with a `length` key whose value is of type `Real`, and the `Real` type is one of our objects. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
```

Comment:
Should extensions.objects be extensions/objects?

Reply:
done